sdk/deepbook-v3/examples/deepbookMarketMaker.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { decodeSuiPrivateKey } from '@mysten/sui.js/cryptography';
+import { getFullnodeUrl, SuiClient } from '@mysten/sui/client';
+import type { Keypair } from '@mysten/sui/cryptography';
+import { Ed25519Keypair } from '@mysten/sui/keypairs/ed25519';
+import type { Transaction } from '@mysten/sui/transactions';
+
+import { DeepBookClient } from '../src/index.js'; // Adjust path according to new structure
+import type { BalanceManager } from '../src/types/index.js';
+
+export class DeepBookMarketMaker extends DeepBookClient {
+	keypair: Keypair;
+	suiClient: SuiClient;
+
+	constructor(
+		keypair: string | Keypair,
```

Comment:
I know this is an example, but if we restrict the input type to just string, then it eliminates any questions related to passing in Keypair

Reply:
There are restrictions to converting privateKey to Keypair (ED25519), so imo shouldn't restrict passing in a string if user has a different Keypair schema